src/shipTemplate.cpp

```diff
@@ -414,6 +418,19 @@ void ShipTemplate::setSpeed(float impulse, float turn, float acceleration)
     impulse_speed = impulse;
     turn_speed = turn;
     impulse_acceleration = acceleration;
+
+    if((impulse == 0.0f) && (impulse_reverse_speed == 0.0f)) // hack supposing we keep existing invariant, to handle template copies
```

Comment:
Does it mean, that it will set reverse speed only if I call `template:setSpeed(0 , 5, 1)`? Because you are comparing against input argument (which I doubt will be a zero). And in constructor, you have `impulse_reverse_speed = 500.0` so I have no idea how this hack could work.

Reply:
It means it's pure madness I don't even understand how I could test templates with this... Just correcting :( So sorry.